subprojects/docs/src/docs/release/notes.md

```diff
@@ -36,6 +36,15 @@ The following are the newly deprecated items in this Gradle release. If you have
 
 ## Potential breaking changes
 
+### Changes in the caching of missing versions
+
+Previously, Gradle would refresh the version list when dynamic version cache expires for repositories that did not contain a version at all.
+From now on, Gradle [will cache forever that a `group:name` was absent](https://github.com/gradle/gradle/issues/4436) from a repository.
+This will impact positively dependency resolution when multiple repositories are defined and dynamic versions are used.
+As before, using `--refresh-dependencies` will force a refresh, bypassing all caching.
+
+If you happen to start provisioning versions to a previously empty - and present in your build - repository, you will need to force a dependencies refresh to have your build take them into account.
```

Comment:
I would remove this sentence. It sounds too dramatic in my mind and does not completely explain when you have to do it. Because you only need to do it when you also use dynamic versions. And when the module was found somewhere else before.

Reply:
Ok, makes sense